src/pal/src/include/pal/sharedmemory.h

```diff
@@ -27,18 +28,18 @@
 #define SHARED_MEMORY_MAX_FILE_NAME_CHAR_COUNT (_MAX_FNAME - 1)
```

Comment:
Could you please also update the comments above? I would probably just start with saying the the following paths are relative to the temporary folder and then put in just the relative parts of the path or use TMP in the paths instead and explain that TMP is platform specific and corresponds to the TEMP_DIRECTORY_PATH or something along these lines.